package.json

```diff
@@ -1,20 +1,25 @@
 {
   "name": "@oclif/test",
   "description": "test helpers for oclif components",
-  "version": "3.2.15",
+  "version": "4.0.1-beta.4",
```

Comment:
wanna remove the prerelease tag? per this validation, it would fail on release (PR targets `main`): 

https://github.com/salesforcecli/github-workflows/blob/a1fcc3038db4e6ab9e505e4129808525bbc6fe19/.github/workflows/create-github-release.yml#L43C1-L48C242

Reply:
That check ensures that you're not manually running `create-github-release` on main and providing prerelease. I don't think it applies to merging PRs to main that have been prereleased